element_interface/intan_loader.py

```diff
@@ -248,40 +240,57 @@ def load_rhs(folder: str, file_expr: str):
             rhs_data["timestamps"] (np.array_like): Relative timestamps in seconds.
     """
 
+    rhs_data = {}
+
+    # Get header
     header_filepath = next(Path(folder).glob("info.rhs"))
     with open(header_filepath, "rb") as fid:
-        header = read_header(fid)
+        rhs_data["header"] = _read_header(fid)
 
+    # Get timestamps
     time_file = next(Path(folder).glob("time.dat"))
 
-    timestamps = (
+    rhs_data["timestamps"] = (
         np.memmap(time_file, dtype=np.int32)
-        / header["frequency_parameters"]["amplifier_sample_rate"]
+        / rhs_data["header"]["frequency_parameters"]["amplifier_sample_rate"]
     )
 
-    rhs_data = dict(header=header, timestamps=timestamps, recordings={})
-
+    # Get data files
     file_paths = Path(folder).glob(file_expr)
-    file_paths = [x for x in file_paths if x.as_posix() != "time.dat"]
 
-    for file_path in file_paths:
-        file_path = file_path.as_posix()
-        if "amp" in file_path:
+    if file_expr == "*":
+        exclude_list = ["time", "info", "Zone.Identifier"]
+
+        file_paths = [
+            file
+            for file in file_paths
+            if not any(string in file.as_posix() for string in exclude_list)
+        ]
+
+    # Get recording data
+    rhs_data["recordings"] = {}
+
+    for file_path in sorted(file_paths):
```

Comment:
@JaerongA This for loop should be run with the condition of `if len(file_paths):`. It is to support the `file_expr` keyword. For instance if file_expr="time.dat", this code will fail because `file_paths` will be an empty list. Let's fix this and merge the PR.

Reply:
@tdincer I thought `file_expre` was intended for selecting data channels only. I've made some revisions to fix that but right now, `timestamps` and `info` will always be read by default. Please feel free to modify if that wasn't your intention.